generate.sh

```diff
@@ -6,9 +6,12 @@ if [ ! -e fhir-parser ]; then
 	git submodule update --init
 fi
 
+pip3 install -r generator-requirements.txt
```

Comment:
@p2 🤔 That makes sense. Does it also apply to [requirements.txt](https://github.com/smart-on-fhir/Swift-FHIR/pull/9/files#diff-ca18249b58faae4bb31643f0b73d12feR15)? 
Goal here was to reduce the generation process to a single command. Is there a better way to do it?